specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/batchInference.json

```diff
@@ -0,0 +1,1712 @@
+        },
+        "link": {
+          "type": "string"
+        }
+      }
+    },
+    "JobOutput": {
+      "description": "Output of a job.",
+      "type": "object",
+      "properties": {
+        "datastoreId": {
+          "description": "Datastore Identifier",
+          "type": "string"
+        },
+        "path": {
+          "description": "Path inside the datastore",
+          "type": "string"
+        }
+      }
+    },
+    "JobEndpoint": {
+      "description": "Job endpoint.",
+      "enum": [
+        "Tracking",
+        "Studio",
+        "Grafana",
+        "Tensorboard"
+      ],
+      "type": "string"
+    },
+    "JobStatus": {
+      "description": "The status of a job.",
+      "enum": [
+        "NotStarted",
+        "Starting",
+        "Provisioning",
+        "Preparing",
+        "Queued",
+        "Running",
+        "Finalizing",
+        "CancelRequested",
+        "Completed",
+        "Failed",
+        "Canceled",
+        "NotResponding"
+      ],
+      "type": "string"
+    },
+    "BatchJob": {
+      "description": "Batch job inference settings.",
+      "type": "object",
+      "properties": {
+        "scaleSettings": {
+          "$ref": "#/definitions/DeploymentScaleSettings"
+        },
+        "datasetId": {
+          "description": "Identifier for input dataset.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Contains resource tags defined as key/value pairs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "Property dictionary. Properties can be added, but not removed or altered.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "errorThreshold": {
+          "format": "int32",
+          "description": "Error threshold, if the error count for the entire input goes above this value,\r\nthe batch inference will be aborted. Range is [-1, int.MaxValue]\r\n-1 value indicates, ignore all failures during batch inference\r\nFor FileDataset count of file failures\r\nFor TabularDataset, this is the count of record failures",
+          "type": "integer"
+        },
+        "partitioningScheme": {
+          "$ref": "#/definitions/BatchPartitioningScheme"
+        },
+        "output": {
+          "$ref": "#/definitions/JobOutput"
+        },
+        "status": {
+          "$ref": "#/definitions/JobStatus"
+        },
+        "interactionEndpoints": {
+          "$ref": "#/definitions/InteractionEndpoints"
+        },
+        "description": {
+          "description": "Datastore Identifier",
+          "type": "string"
+        }
+      }
+    },
+    "BatchJobResource": {
+      "description": "Azure Resource Manager resource Envelope",
+      "allOf": [
+        {
+          "$ref": "machineLearningServices.json#/definitions/Resource"
```

Comment:
This inherited "resource" definiton contains "tags" and "location" and "identity" which is not appropriate for this resource, correct?  You are also re-defining "id", "name", and "type" below